android/app/src/main/res/values/strings.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <resources>
   <!-- Shown on app info, app switcher, etc -->
-  <string name="app_name">COVID Safe Paths</string>
+  <string translatable="false" name="app_name">COVID Safe Paths</string>
   <!-- Shown below icon on launcher only -->
-  <string name="app_name_short">Safe Paths</string>
+  <string translatable="false" name="app_name_short">Safe Paths</string>
   <!-- Channel label for notifications shown in App > Notifications > Safe Paths -->
-  <string name="notification_channel_description">Handle all local notifications</string>
+  <string name="notification_channel_description">Safe Paths notifications</string>
```

Comment:
is this change reflected in all of the translated versions?

Reply:
It will fall back to the english until we have translations. And yes, translators will see the new english base value